packages/url_launcher/url_launcher_ios/example/ios/RunnerTests/URLLauncherTests.swift

```diff
@@ -0,0 +1,141 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+import XCTest
+
+@testable import url_launcher_ios
+
+final class URLLauncherTests: XCTestCase {
+  private var plugin: FLTURLLauncherPlugin!
+  private var launcher: FakeLauncher!
```

Comment:
Why did the test class become stateful in this migration? Stateful test fixtures [are generally an anti-pattern](https://abseil.io/tips/122), and making the object under test stateful is especially problematic.